### PR TITLE
Cc mempool2

### PR DESCRIPTION
**Pull request by chengchen666**
only commit: 22c9a0daaddd04b2502a8f6c34b8d31304adef1b is needed, other commits come from dev3 branch.

### CODE REVIEW

File: qkernel/src/syscalls/sys_proxy.rs [hunk 1]
```diff
@@ -819,7 +819,7 @@ fn CudaMemcpyAsync(
     kind: CudaMemcpyKind,
     stream: u64,
 ) -> Result<i64> {
-    // error!("CudaMemcpyAsync: count:{}, kind{}", count, kind);
+    // error!("CudaMemcpyAsync: count:{:x}, kind{}, src:{:x}, dst:{:x}", count, kind, src, dst);
```
Comment (chl337) [inline qkernel/src/syscalls/sys_proxy.rs:822]: If it is not needed, please consider removing commented out code (same for below).

File: qvisor/src/vmspace/xpu/cuda_mem_manager.rs [hunk 2]
```diff
@@ -0,0 +1,276 @@
+use crate::vmspace::{CUDA_MEMCPY_DEVICE_TO_HOST, CUDA_MEMCPY_HOST_TO_DEVICE};
+use crate::xpu::cuda_api::*;
+use std::collections::BTreeMap;
+use std::sync::atomic::{AtomicU32, Ordering};
+use cuda_runtime_sys::cudaStream_t;
+
+pub const ONE_TB: u64 = 1 << 40; //0x10_000_000_000;
```
Comment (chl337) [inline qvisor/src/vmspace/xpu/cuda_mem_manager.rs:7]: Duplication of [declaration](https://github.com/QuarkContainer/Quark/blob/922cfe712075a0ce41024cfe1a566724fb2da79f/qlib/linux_def.rs#L2953).

**Comment by shrik3** [comment]
> only commit: [22c9a0d](https://github.com/QuarkContainer/Quark/commit/22c9a0daaddd04b2502a8f6c34b8d31304adef1b) is needed, other commits come from dev3 branch.

#1240 it's already merged, you may rebase against current main now.

**Comment by chengchen666** [comment]
Yes, will create a new branch and rebase, then create another PR later.

**Comment by chengchen666** [comment]
moved to #1246

**Pull request closed without merging**